### ISSUE TITLE
Error in clean_wqdata

### ISSUE THREAD

**Issue by HeatherGranger** [bcgov/shinyrems#49]
`Error in wqbc::clean_wqdata(x = x, by = by, max_cv = max_cv, sds = sds,  : 
  unused argument (FUN = FUN)`

**Comment by sebdalgarno** [comment]
Thanks for this @HeatherGranger! This is an installation issue. I've just updated the readme and installation instructions.

Since the app now works from development version of rems and wqbc these also both have to be installed by the user prior to installing shinyrems.

Can you try:

```
remotes::install_github("bcgov/rems")
remotes::install_github("bcgov/wqbc#117)
remotes::install_github("bcgov/shinyrems")
# restart R session
```

and confirm that that solves the problem?

**Comment by HeatherGranger** [comment]
is `run_ems_app` still the function to open the app?

**Comment by sebdalgarno** [comment]
sorry, the code should be:

```
```
remotes::install_github("bcgov/rems")
remotes::install_github("bcgov/wqbc")
remotes::install_github("bcgov/shinyrems")
# restart R session
```

**Comment by sebdalgarno** [comment]
yes run_ems_app() is still the function to open the app

**Comment by HeatherGranger** [comment]
I'm getting `Error: 'run_ems_app' is not an exported object from 'namespace:shinyrems'

**Comment by sebdalgarno** [comment]
oh boy its the end of the day!
I'm sorry it should be:

```
remotes::install_github("bcgov/rems")
remotes::install_github("bcgov/wqbc")
remotes::install_github("poissonconsulting/shinyrems")
# restart R session
```

(grab shinyrems from poissonconsulting not the old bcgov repo)

**Comment by HeatherGranger** [comment]
I hear ya! I'm working from home with a toddler. 
Works great, thanks. Shall we review now? I see the guideline option is in there now. Is it just pulling in the aquatic life guidelines or the entire new limits look up table?

**Comment by sebdalgarno** [comment]
Jessica and I discussed this a bit on Slack today.
Basically the app as it currently is is calculating guidelines from the old limits table. The new limits table (~ 90 freshwater limit records have been added and some limit equations updated) will be added to wqbc once this pull request is approved by @ateucher (https://github.com/bcgov/wqbc/pull/117). I don't anticipate any issues when the app moves to the new limits table (it is simply running calc_limits and many tests already in wqbc test that the new limits table works). so really it is up to you guys whether to review now or wait until PR is approved
